barretenberg/cpp/src/barretenberg/constants.hpp

```diff
@@ -13,6 +13,8 @@ static constexpr uint32_t CONST_PG_LOG_N = 20;
 
 static constexpr uint32_t CONST_ECCVM_LOG_N = 16;
 
+static constexpr uint32_t NUM_OP_QUEUE_COLUMNS = 4;
```

Comment:
Why the new constant?

Reply:
thought it's more clear but we could do without tbf